### PR TITLE
NPE on ServerRequestQueue methods

### PR DESCRIPTION
**Pull request by Sarkar**
@qinweigong @aaustin @shortstuffsushi

### CODE REVIEW

File: Branch-SDK/src/io/branch/referral/ServerRequestQueue.java [hunk 1]
```diff
@@ -156,8 +156,10 @@ public ServerRequest removeAt(int index) {
 	public boolean containsClose() {
 		synchronized(queue) {
             for (ServerRequest req : queue) {
-                if (req.getTag().equals(BranchRemoteInterface.REQ_TAG_REGISTER_CLOSE)) {
-                    return true;
+                if (req != null) {
+                    if (req.getTag().equals(BranchRemoteInterface.REQ_TAG_REGISTER_CLOSE)) {
```
Comment (qinweigong) [inline Branch-SDK/src/io/branch/referral/ServerRequestQueue.java]: can combine these two lines into  if (req != null && req.getTag().equals....)

Comment (Sarkar) [inline Branch-SDK/src/io/branch/referral/ServerRequestQueue.java]: Java has lazy evaluation, good call

**Comment by qinweigong** [comment]
:+1:

**Comment by aaustin** [comment]
:+1:

**Pull request merged**